Create ui-c8y-1022-24-0-extended-provisioned-certificates-data-grid-with-new-columns-and-aligned-expiration.md (#3665)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-24-0-extended-provisioned-certificates-data-grid-with-new-columns-and-aligned-expiration.md

---
date: ""

> title: 'Provisioned certificates data grid extended 

Reviewer: ```suggestion
title: Provisioned certificates data grid extended 
```

> title: 'Provisioned certificates data grid extended with "Valid from" column'

Reviewer 2: ```suggestion
title: 'Provisioned certificates data grid extended 
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64414
version: 1022.24.0
---

> To provide a better overview of provisioned certificates, the "Provisioned certificates" data grid has been extended. A new "Valid from" column has been added which shows the start date of the certificate's validity. Additionally, the expiration highlight styles in this data grid have been aligned with the ones already used in the "Trusted certificates" view for a more consistent user experience. These changes allow users to see the full validity period of certificates at a glance and quickly identify certificates that are about to expire.

Reviewer 2: ```suggestion
To provide a better overview of provisioned certificates, the **Provisioned certificates** data grid has been extended. A new "Valid from" column has been added, which shows the start date of the certificate's validity. Additionally, the expiration highlight styles in this data grid have been aligned with the ones already used in the **Trusted certificates** view for a more consistent user experience. These changes enable users to view the full validity period of certificates at a glance and quickly identify those that are about to expire.
```
